go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/kubernetes-incubator/external-storage v0.20.4-openstorage-rc7
 	github.com/libopenstorage/autopilot-api v1.3.0
 	github.com/libopenstorage/cloudops v0.0.0-20230220114907-3e63dce1b413
-	github.com/libopenstorage/openstorage v9.4.47+incompatible
+	//github.com/libopenstorage/openstorage v9.4.47+incompatible
```

Comment:
Remove it if its not needed, instead of commenting this out, though I still see it gets added as part of `require` on line 152

Reply:
Done. removed commented line and kept the required line 152. Yes, we need github.com/libopenstorage/openstorage v9.4.47+incompatible line in go.mod.